src/archive.rs

```diff
@@ -282,6 +282,7 @@ impl<'a, R: Read> Iterator for Entries<'a, R> {
     }
 }
 
+#[allow(unused_assignments)]
```

Comment:
Hopefully we can drop this now

Reply:
No, rustc 1.87.0-nightly still complains.